src/check_bgp_protocols.py

```diff
@@ -67,6 +67,16 @@ def check_birdc_peers(path):
     if not protocols:
         return (ExitCodes.critical, 'Could not retrieve BIRD\'s output')
 
+    # Here we match the BGP protocol lines from BIRD.
+    # Below is an example of how they look like:
+    # direct1  Direct   master   up     2020-02-27
+    # static_bgp Static   master   up     2020-02-21
+    # b_firstroot BGP      master   up     2020-02-28  Established
+    # b_lb     BGP      master   start  18:43:46    Active        Socket: conn
+    # b_af     BGP      master   up     2020-02-29  Established
+    # b_home   BGP      master   down   2020-02-29
```

Comment:
I just need all possible different lines. With those lines given, we can reduce it to:
```suggestion
    # direct1  Direct   master   up     2020-02-27
    # b_firstroot BGP      master   down     2020-02-28  Established
    # b_lb     BGP      master   start  18:43:46    Active        Socket: conn
```